.github/workflows/bun-support.yml

```
@@ -0,0 +1,83 @@
+name: Bun Support
+#on:
+#  schedule:
+#    - cron: 0 0 * * *
+#  workflow_dispatch:
+
+on:
+  push:
+    branches: [master, develop]
+    tags: ['*']
+  pull_request:
+    types: [opened, reopened, synchronize]
+  workflow_dispatch:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - uses: oven-sh/setup-bun@v1
```

```suggestion
      - uses: oven-sh/setup-bun@v1
        with:
          bun-version: 1.0.24
```
Might want to pin the version so there is no ambiguity if a pr caused a regression or bun version changed it.

Alternatively consider just echoing the bun version in the next step of this test so if you are debugging a workflow you know exactly which version of bun it used